cypress/cucumber-json/menu.cucumber.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
This looks like a test report…

Reply:
Removed and added the directory to .gitignore